recipe/meta.yaml

```diff
@@ -15,26 +15,24 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 8897491277ce80a1ea49306552be778ece4d0bf8fad3373388936554c32bccd8
+  sha256: 193055dbee8701ad62d5471c5287b092ce53bb77f7f810ec68bf96816f6acaa2
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  skip: True  # [py<36]
+  skip: True  # [py<38]
   skip: True  # [linux and (s390x or ppc64le)]
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
-    - setuptools
-    - wheel
   run:
-    - python
+    - python >=3.6
     - matplotlib-base
     - pyqtgraph
     - anyqt >=0.1.0
-    - orange-canvas-core >=0.1.20,<0.2a
+    - orange-canvas-core >=0.1.27,<0.2a
```

Comment:
be sure to update this if you go for v4.20.0

Reply:
Thanks a lot for all the useful tips @danpetry!

Reply:
This was only if you updated to v4.20, which you aren't - it should therefore stay at 0.1.27 to reflect the dependencies of the version you're actually packaging

Reply:
@boldorider4 ping - this could be the reason for the build errors now (I only checked linux-64)

Reply:
The error was due to the build environment not automatically being checked out from scratch upon every commit.